zuul.d/jobs.yaml

```diff
@@ -1087,6 +1087,8 @@
     description: |
       Run integration tests of spark of master with k8s cluster deployed by kubeadm
     run: playbooks/spark-integration-test-kubeadm-k8s/run.yaml
+    vars:
+      k8s_log_dir: '{{ ansible_user_dir }}/workspace/logs/kubernetes'
```

Comment:
Whether we can collect kubernetes services log to here, like: apiserver.log?

Reply:
@liu-sheng sorry, my fault, you collect all of kube-system namespace pod logs, that should include apiserver pod logs, I got it